src/sqlancer/tidb/TiDBBugs.java

```diff
@@ -55,7 +52,7 @@ public final class TiDBBugs {
     public static boolean bug35652 = true;
 
     // https://github.com/pingcap/tidb/issues/38295
-    public static boolean bug38295 = true;
+    public static boolean bug38295;
```

Comment:
Do you think we should remove the code that is executed when this bug is set to `true` as well, or should we keep it in to serve as documentation?

Reply:
In fact, I find the bug switch set as false if the bug is fixed. I think it may be a coding habit of sqlancer. 

https://github.com/sqlancer/sqlancer/blob/102ec94c87f281b66a1ae34d0c75a3b98f5ebe03/src/sqlancer/tidb/TiDBBugs.java#L6-L13

Reply:
https://github.com/pingcap/tidb/issues/38295 has been fixed. I think we can remove the code and limit. Although we can find the ``` assertion failed```, it is not the same problem.

Reply:
Can I remove all false bug switches if we think it is fixed?

Reply:
I find the problem in the CI. Let me study in depth. >。<

Reply:
> Can I remove all false bug switches if we think it is fixed?

Yes, I think we could remove them, which would reduce also maintenance costs.

Reply:
I figured that some implementations keep the flags and link to the relevant bug reports in the code (for documentation purposes?) as you mentioned - for me, either way is fine.